tests/testerina-integration-test/src/test/resources/project-based-tests/code-coverage-report-test/line-insert-and-remove-test/main.bal

```diff
@@ -0,0 +1,18 @@
+import samjs/package_comp_plugin_code_modify_add_remove_function as _;
+
+public function main() {
+    _ = intAdd(1,2);
+}
+
+function baz () {
+
+}
+
+function qux () {
+
```

Comment:
Is this additional new line intentional?

Reply:
Yes. I have added the additional new lines so that the #lines removed will be greater than the # new lines added.